src/api/auth.ts

```diff
@@ -0,0 +1,13 @@
+import * as crypto from 'crypto';
+
+export function generateRandomString(length: number) {
+	const characterSet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
```

Comment:
Better to make this as an `export const CHARACTER_STE = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789" as const`, then import it from your `auth.test.ts` file